telemetry/definitions/commonDefinitions.json

```diff
@@ -6351,6 +6351,10 @@
                 {
                     "type": "result",
                     "required": true
+                },
+                {
+                    "type": "version",
+                    "required": false
```

Comment:
What would happen if this is set to `"required": true`? Seems like it would be an important field to have

Reply:
`version` isn't an applicable field to all "modules"; it currently only has relevance to the "webview2" module. we wouldn't want this to be a required field for all modules

Reply:
This would make it so that every module would need to specify this field. We left it as optional since currently only the RuntimeInstaller module has a version that it would pass through.

Reply:
Got it, thank you for the context!